### DISCUSSION TITLE
在Z.O.E产品帖下发布评论即可有机会赢取一架ADF-01型战斗机!

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#3065] [其他]
![屏幕截图 2024-08-18 233804](https://image.gcores.com/f0431133-53b8-4e4e-8b0a-5b20fbe97d92.png?x-oss-process=image/quality,q_90/format,webp/watermark,image_d2F0ZXJtYXJrLnBuZw,g_se,x_10,y_10)
# 从即日起至八月21日 在Z.O.E产品发布帖下发表评论即可有机会赢取一架ADF-01型战斗机!
# *发表诋毁我司或者无意义评论将不视为参加
# *本帖及其他非Z.O.E产品帖下评论不视为参加

**Comment by lllliufeng** [comment]
我我我，选我助力新艾利都走上强军之路（

**Comment by LT-UT** [comment]
我，老顾客了，选<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >我
